### DISCUSSION TITLE
How to change only the top left color of sidebar

### DISCUSSION THREAD

**Discussion by noblemfd** [ColorlibHQ/AdminLTE#4239] [Q&A] [answered]
I am using AdminLTE-3.2.0-rc. 

1. I want to change the top-left backcolor of the white or any other color.
2. Also change the font color from white to black or other color since the back color will be white

![sidebar](https://user-images.githubusercontent.com/34077129/151963009-b45a11e9-ece7-4033-a332-742a4d1b83e8.PNG)

I tried to change

    <aside class="main-sidebar sidebar-dark-primary elevation-4">

into

    <aside class="main-sidebar sidebar-white-primary elevation-4">

but it change the whole sidebar color.

How do I achieve this?

Thanks

**Comment by REJack** [comment]
Simply add `.bg-white` or any other bg-class to `.brand-link`.

**Comment by dfsmania** [comment, marked as answer]
For example, if you add `bg-teal` class to the `.brand-link`  element, as in the next example:

```html
<a href="index3.html" class="brand-link bg-teal">
    <img src="dist/img/AdminLTELogo.png" alt="AdminLTE Logo" class="brand-image img-circle elevation-3" style="opacity: .8">
    <span class="brand-text font-weight-light">AdminLTE 3</span>
</a>
```
You will get...

![Screenshot 2022-05-11 at 18-39-21 AdminLTE 3 Dashboard](https://user-images.githubusercontent.com/63609705/167952453-c51adcec-0bc2-4ea3-937c-e3c73b46f46b.png)
